### PR TITLE
22310 complete page

### PR DESCRIPTION
**Pull request by nnorbert**
Complete page

### CODE REVIEW

File: apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  if (isInactivePublicServant) {
+    return redirect("/admin/inactivePublicServant", RedirectType.replace);
+  }
+
+  if (isPublicServant) {
+    return redirect("/admin/journeys", RedirectType.replace);
+  }
+
+  const summary = await getJourneySummary(journeyId, runId);
```
Comment (alfonsograziano) [inline apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx:43]: Non-blocker: Cannot we call directly `client.getJourneySummary` within the RSC without creating the server action?

Comment (nnorbert) [reply apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx:43]: We can, only we used this technique in the other pages as well.

**Review by alfonsograziano — DISMISSED** [review]
LGTM - Just left a couple of comments :)

File: apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx (hunk 1 above)
Comment (alfonsograziano) [reply apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx:43]: Maybe is worth simplifying it if we can :) But again, not a blocker for me

Comment (nnorbert) [reply apps/integrator/app/[locale]/(hosted)/journey/[journey]/complete/[runId]/page.tsx]: Done!

File: apps/intg-api/src/routes/journeys/index.ts [hunk 2]
```diff
@@ -350,6 +350,15 @@ export default async function journeys(app: FastifyInstance) {
       stepData: {},
     });
 
+    const completeStep = await app.journeySteps.createStep({
+      journeyId: journey.id,
+      stepType: "complete",
+      stepData: {
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/intg-api/src/routes/journeys/index.ts:366]: Done!

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**